tests/mocks/auth.mock.js

```diff
@@ -0,0 +1,10 @@
+const mockedVerifyToken = (req, res, next) => {
+  req.user = {
+    id: "test-user-id",
+  };
+  next();
+};
+
+module.exports = {
```

Comment:
```suggestion
module.exports = mockedVerifyToken
```
`mockedVerifyToken`이 객체안에 있을 필요가 없어 보입니다!